app/screens/ScheduleScreen/ScheduleScreen.tsx

```diff
@@ -277,7 +277,11 @@ export const ScheduleScreen: React.FC<TabScreenProps<"Schedule">> = () => {
           />
         )}
       </View>
-      <ScheduleDayPicker {...{ scrollX, onItemPress, schedules, selectedSchedule }} />
+      <ScheduleDayPicker
+        {...{ scrollX, onItemPress }}
+        scheduleDates={schedules.map((s) => parseDate(s.date))}
+        selectedScheduleDate={parseDate(selectedSchedule.date)}
```

Comment:
I would eventually like to push the date parsing down into the data layer. That makes working with the data layer easier in that I don't have to guess whether I need to parse it or not. It's a date.

Reply:
I agree on this point because of the whole date should be in PT and all